quoridor-app/app/component/board.js

```diff
@@ -22,23 +23,31 @@ const generateKey = (type, position) => {
   return `${type}:${position.column}${position.row}`;
 };
 
-const Board = ({ squares, pawns }) => {
-  if (!squares) {
+const isSquareDisabled = (moves, position) => {
+  const exist = moves.filter(
```

Comment:
you can use `moves.findIndex` here and check that the result is equal to `-1`
It is better than building an empty list